Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Tools/UtilsTH.hs

```diff
@@ -0,0 +1,29 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module Tools.UtilsTH (module Reexport, module Tools.UtilsTH) where
```

Comment:
Don't like this path. Maybe `Tools.Beam.UtilsTH` or `Storage.Beam.Lib.UtilsTH`?

Reply:
There's a Lib Folder already in src i guess